clang/lib/CConv/CConv.cpp

```diff
@@ -109,14 +109,15 @@ ArgumentsAdjuster getIgnoreCheckedPointerAdjuster() {
     bool HasAdjuster = false;
     for (size_t i = 0, e = Args.size(); i < e; ++i) {
       StringRef Arg = Args[i];
-      AdjustedArgs.push_back(Args[i]);
+      AdjustedArgs.push_back(Arg);
       if (Arg == "-fcheckedc-convert-tool") {
         HasAdjuster = true;
-        break;
```

Comment:
This looks like a bugfix. Could it be the reason the `-nostdinc` wasn't working?

Reply:
Maybe! I can try testing it on a file by removing the stddef call, changing to int to cause the failure and trying again.